pkgs/development/interpreters/python/python-packages-base.nix

```diff
@@ -19,9 +19,9 @@ let
       ff = f origArgs;
       overrideWith = newArgs: origArgs // (if pkgs.lib.isFunction newArgs then newArgs origArgs else newArgs);
     in
-      if builtins.isAttrs ff then (ff // {
+      if builtins.isAttrs ff then ff.overrideAttrs (previousAttrs: { passthru = (previousAttrs.passthru or {}) // {
```

Comment:
For performance, it'd be better to `composeExtensions` (or similar) this to the `origArgs`. That way we don't evaluate all python packages twice.